CHANGELOG-3.5.md

```diff
@@ -119,6 +119,7 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
   - Previously, server restore fails if it had crashed after persisting raft hard state but before saving snapshot.
   - See https://github.com/etcd-io/etcd/issues/10219 for more.
 - Improve logging around snapshot send and receive.
+- Fix [crc mismatch crash bug](https://github.com/etcd-io/etcd/pull/11924).
```

Comment:
`Add missing CRC checksum check in WAL validate method otherwise causes panic https://github.com/etcd-io/etcd/issues/11918`?

Reply:
updated. thanks. @gyuho